src/client/components/pages/entity-revisions.js

```diff
@@ -24,14 +26,28 @@ const Col = bootstrap.Col;
 const ListGroup = bootstrap.ListGroup;
 const ListGroupItem = bootstrap.ListGroupItem;
 
+/**
+* This is an extended class of the React Component to
```

Comment:
"This class is derived from the React Component base class and renders..." would be better, I think.